create_single_match_data.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+
+def create_single_match_data(gap_df):
+    a_only = gap_df.drop(["B", "B-offset", "B-coref"], axis=1)
+    part1 = a_only.rename(columns={"A": "Name", "A-offset": "Name-offset", "A-coref": "Coref"})
```

Comment:
Shouldn't we keep `B` and `B-offset` here (and vice versa for A in the `b_only` group) as `other_name` and `other_offset` ?

Reply:
I can see keeping the second one in there as other maybe, but shouldn't all the data have the same column names so that we can analyse them together as one frame with 8000 rows?

Reply:
Ah, I misunderstood. Done.